core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -187,6 +188,14 @@ default void waitTopicDeletion(String topic) throws InterruptedException {
         waitForTopic(topic, 0);
     }
 
+    default void consistentMetadata() throws InterruptedException {}
```

Comment:
Could you please offer implementation for it? I assume it has following behaviors.

1. return if there is no quorum controller
2. `controllerOffset` should be the "largest offset"

BTW, how about renaming it to `waitForMeatdataSync`
WDYT?

Reply:
Hi @chia7712 , thanks for review! I have updated this PR.